packages/app/components/Nation/Nation.tsx

```diff
@@ -7,6 +7,8 @@ import Territories from './Territories';
 import withGoToWelcome, {
   WithGoToWelcomeProps
 } from './decorators/withGoToWelcome';
+import { compose } from 'recompose';
+import { MapProps } from '@chronoscio/app/pages/map';
```

Comment:
Generally for imports, absolute imports go in the top block, relative ones in the bottom block. I'll have a look if tslint has such a rule, if yes I'll add it.

Reply:
Right, I think I mostly followed this but some of the vscode auto imports ended up at the bottom.